src/wp-admin/js/inline-edit-tax.js

```diff
@@ -137,27 +197,47 @@ inlineEditTax = {
 				}
 			}
 		);
+
 		// Prevent submitting the form when pressing Enter on a focused field.
 		return false;
 	},
 
+	/**
+	 * Closes the quick edit form.
+	 *
+	 * @this inlineEditTax
+	 * @memberof inlineEditTax
+	 */
 	revert : function() {
 		var id = $('table.widefat tr.inline-editor').attr('id');
 
 		if ( id ) {
 			$( 'table.widefat .spinner' ).removeClass( 'is-active' );
 			$('#'+id).siblings('tr.hidden').addBack().remove();
 			id = id.substr( id.lastIndexOf('-') + 1 );
+
 			// Show the taxonomy row and move focus back to the Quick Edit link.
 			$( this.what + id ).show().find( '.editinline' ).focus();
 		}
 	},
 
+	/**
+	 * Retrieves the ID of the term of the element inside the table row.
+	 *
+	 * @memberof inlineEditTax
+	 *
+	 * @param {HTMLElement} o An element within the table row or the table row itself.
+	 * @returns {string} The ID of the term based on the element.
+	 */
 	getId : function(o) {
 		var id = o.tagName === 'TR' ? o.id : $(o).parents('tr').attr('id'), parts = id.split('-');
+
 		return parts[parts.length - 1];
 	}
 };
 
-$(document).ready(function(){inlineEditTax.init();});
+$(document).ready(function(){
+	inlineEditTax.init();
```

Comment:
As trivial as this change is, it's what'd be considered a refactor, so don't do this in a docs change.